.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+  We ❤️  Pull Requests! :-)
+
+  This template is not a must to use, however it would help us a lot to speed up
+  your pull request review if you would be so kind and compile it with your code
+  submission an dyour Pull Request!
+  If you remove or skip this template, you'll make the Elders of RISC OS sad and 
+  the mighty King Arthur might rise his ARM against you!!!! you're warned :)
```

Comment:
```suggestion
  the mighty King Arthur might raise his ARM against you!!! You have been warned :)
```

Reply:
In addition, maybe an exclamation-mark after "warned". I'm not how well received this flippant style will be with the community, but you probably know what sense of humour the would-be contributors have better than I do. ;)